merkle/sparse_merkle_tree_test.go

```diff
@@ -281,3 +285,82 @@ func TestRootAtRevisionCatchesNonRootNode(t *testing.T) {
 		t.Fatalf("Attempt to retrieve root with corrupt node did not result in error: %v", err)
 	}
 }
+
+func TestInclusionProofForNullEntryInEmptyTree(t *testing.T) {
+	const rev = 100
+	r, tx := getSparseMerkleTreeReaderWithMockTX(rev)
+	tx.On("GetMerkleNodes", int64(rev), mock.AnythingOfType("[]storage.NodeID")).Return([]storage.Node{}, nil)
```

Comment:
I thought you could only use AnythingOfType in asserts. Are you sure the mocks are all wired up correctly?